OrbitCaptureGgpClient/CMakeLists.txt

```diff
@@ -2,27 +2,40 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-project(OrbitCaptureGgpClient)
-
-add_executable(OrbitCaptureGgpClient main.cpp)
+project(OrbitCaptureGgpClientLib)
+add_library(OrbitCaptureGgpClientLib STATIC)
 
-target_compile_options(OrbitCaptureGgpClient PRIVATE 
+target_compile_options(OrbitCaptureGgpClientLib PRIVATE 
         ${STRICT_COMPILE_FLAGS})
 
-target_include_directories(OrbitCaptureGgpClient PUBLIC 
+target_include_directories(OrbitCaptureGgpClientLib PUBLIC 
         ${CMAKE_CURRENT_LIST_DIR}/include)
 
-target_include_directories(OrbitCaptureGgpClient PRIVATE 
+target_include_directories(OrbitCaptureGgpClientLib PRIVATE 
         ${CMAKE_CURRENT_LIST_DIR})
 
-target_sources(OrbitCaptureGgpClient PUBLIC
+target_sources(OrbitCaptureGgpClientLib PUBLIC
         include/OrbitCaptureGgpClient/OrbitCaptureGgpClient.h)
 
-target_sources(OrbitCaptureGgpClient PRIVATE
-        main.cpp
+target_sources(OrbitCaptureGgpClientLib PRIVATE
         OrbitCaptureGgpClient.cpp)
 
-target_link_libraries(OrbitCaptureGgpClient PUBLIC 
+target_link_libraries(OrbitCaptureGgpClientLib PUBLIC 
         OrbitBase
         OrbitProtos)
 
+if(WIN32)
```

Comment:
You can remove these defines since we are not compiling for windows anyway.

Reply:
You are right, I missed this. I have removed it. Thanks!